contracts/optimistic-ethereum/libraries/rlp/Lib_RLPReader.sol

```diff
@@ -292,6 +292,42 @@ library Lib_RLPReader {
         );
     }
 
+    /**
+     * Reads an RLP uint64 value into a uint64.
+     * @param _in RLP uint64 value.
+     * @return Decoded uint64.
+     */
+    function readUint64(
+        RLPItem memory _in
+    )
+        internal
+        pure
+        returns (
+            uint64
+        )
+    {
+        return uint64(uint256(readBytes32(_in)));
```

Comment:
Can we add test cases alongside the existing test JSON for `Lib_RLPReader` just to keep up the coverage?  Looks fine but still good practice.